Flabbergasted-Flamingos/bangazon-commerce-app
# Rb product detail

# Description
Product detail now works

## Related Ticket(s)
#10 

## Problem to Solve
Users can click on product in the list view and it takes them to product detail view

## Proposed Changes
This affects urls, views and templates

## Expected Behavior
When you view the /products page all the products should have a hyperlink, when you click on an individual link it will take you to the product page

## Steps to Test Solution

1. 
```
git fetch --all
git checkout rb_product_detail
python manage.py runserver
```
go to localhose/products
click on a product
make sure it goes to the detail page
